bin/start-error-log

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+rm -rf import_log.txt
+rm -rf error_log.txt
+
+node --max_old_space_size=6000 import.js  &> import_log.txt && cat import_log.txt | awk '$1 ~ /error/' >> error_log.txt
+
+echo -e "..............................."
+echo -n 'The total number of errors are: ' & wc -l error_log.txt
+echo -e "..............................."
+echo -n 'Total Error type: invalid_shape_exception [400] ' & grep "invalid_shape_exception" error_log.txt | wc -l
+echo -n 'Error type: invalid_shape_exception [400]: Provided shape has duplicate consecutive coordinates at: ' & grep "Provided shape has duplicate consecutive coordinates at:" error_log.txt | wc -l
+echo -n 'Error type: invalid_shape_exception [400]: Cannot determine orientation: ' & grep "Cannot determine orientation:" error_log.txt  | wc -l
+echo -n 'Error type: invalid_shape_exception [400]: Self-intersection at or near point: ' & grep "Self-intersection at or near point" error_log.txt  | wc -
+echo -e "..............................."
+echo -n 'Total Error type: mapper_parsing_exception [400]: Unable to Tessellate shape: ' & grep "Unable to Tessellate shape" error_log.txt  | wc -l
+echo -e "..............................."
+echo -n 'Total Error type: transaction error reached max retries ' & grep "transaction error reached max retries" error_log.txt | wc -l
+echo -n 'Total Error type: invalid resp from es bulk index operation: ' & grep "invalid resp from es bulk index operation" error_log.txt  | wc -l
+echo -n 'Total Error type: esclient error message=Request Entity Too Large, status=413: ' & grep "Request Entity Too Large"   error_log.txt  | wc -l
```

Comment:
This is interesting output, but I don't think this is how we'd like to implement it. Over in the [OpenAddresses importer](https://github.com/pelias/openaddresses/blob/d89f586df74ada814bbc379e9d20dca7bf4eeae3/lib/streams/documentStream.js#L38) we pass around a `stats` object that can record counts of various interesting events.

If you'd like to implement that, it would be fantastic, but let's do it in a separate PR.